product_code/lagom/authentication_service/impl/src/main/scala/de/upb/cs/uc4/authentication/impl/actor/AuthenticationState.scala

```diff
@@ -20,7 +21,17 @@ case class AuthenticationState(optEntry: Option[AuthenticationEntry]) {
    */
   def applyCommand(cmd: AuthenticationCommand): ReplyEffect[AuthenticationEvent, AuthenticationState] =
     cmd match {
-      case SetAuthentication(user, replyTo) => Effect.persist(OnSet(user)).thenReply(replyTo) { _ => Accepted }
+      case SetAuthentication(user, replyTo) =>
+
+        val validationErrors = user.validate.map(error => SimpleError("authUser." + error.name, error.reason))
```

Comment:
"authUser." is only the correct prefix when used in addUser, not in changePassword